beautiful-and-accessible-drag-and-drop-with-react-beautiful-dnd/notes/04-react-reorder-a-list-with-react-beautiful-dnd.md

```diff
@@ -248,6 +243,7 @@ export default class Task extends React.Component {
   }
 }
 ```
+
 </TimeStamp>
```

Comment:
empty line both _before_ closing and _after_ opening `TimeStamp` is necessary in MDX@1